### PR TITLE
manual: test IM proxy gRPC connection metrics

### PR DESCRIPTION
**Pull request by c3y1huang**
https://github.com/longhorn/longhorn/issues/4038
https://github.com/longhorn/longhorn/issues/2821

### CODE REVIEW

File: docs/content/manual/release-specific/v1.3.0/test-grpc-proxy.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+## Test gRPC proxy
+
+**Given** Longhorn exist in the cluster.
+**And** Monitoring stack exist in the cluster.
+
+**When** Execute `longhorn_instance_manager_proxy_grpc_connection` in Prometheus UI.
+**Then** Metric data shows in Prometheus UI.
+
+**When** Monitor `longhorn_instance_manager_proxy_grpc_connection` in Grafana UI Panel.
+*And* Run full automation regression.
```
Comment (innobead) [inline docs/content/manual/release-specific/v1.3.0/test-grpc-proxy.md:18]: For QA, it should be good enough by running sanity.

**Review by innobead — DISMISSED** [review]

File: docs/content/manual/release-specific/v1.3.0/test-grpc-proxy.md (hunk 1 above)
Comment (c3y1huang) [reply docs/content/manual/release-specific/v1.3.0/test-grpc-proxy.md:18]: I suggest running full regression a least one time.

File: docs/content/manual/release-specific/v1.3.0/test-grpc-proxy.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: Test gRPC proxy
```
Comment (innobead) [inline docs/content/manual/release-specific/v1.3.0/test-grpc-proxy.md:2]:
```suggestion
title: Test IM Proxy connection metrics
```

File: docs/content/manual/release-specific/v1.3.0/test-grpc-proxy.md (hunk 1 above)
Comment (c3y1huang) [reply docs/content/manual/release-specific/v1.3.0/test-grpc-proxy.md:18]: I've already run full regression last night with [PR#1379](https://github.com/longhorn/longhorn-manager/pull/1379#issuecomment-1155956950) and seems no major leak except 1 connection that I am not sure of.

After talking to @innobead , we agree @longhorn/qa can do `coretest` so we don't need to duplicate most of the testings.

**Review by innobead — APPROVED** [review]

**Pull request merged**